README.rst

```diff
@@ -92,7 +92,7 @@ demonstration of implementing the "Happy Eyeballs" algorithm in an
 older library versus Trio.
 
 **Cool, but will it work on my system?** Probably! As long as you have
-some kind of Python 3.7-or-better (CPython or the latest PyPy3 are
+some kind of Python 3.8-or-better (CPython or currently maintaned versions of PyPy3 are
 both fine), and are using Linux, macOS, Windows, or FreeBSD, then Trio
```

Comment:
Mind linking to https://doc.pypy.org/en/latest/faq.html#which-python-versions-does-pypy-implement with a `[text](link)` thing here?

Reply:
The FAQ entry currently has a broken link but that's just a simple issue to fix: I haven't made an issue about it yet though.